src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PDB/MethodDebugInfo.Native.cs

```diff
@@ -135,18 +130,38 @@ public override int GetHashCode()
                         out importRecordGroups,
                         out externAliasRecords);
 
-                    ReadCSharpNativeCustomDebugInfo(
-                        symReader,
-                        methodToken,
-                        methodVersion,
-                        allScopes,
-                        out hoistedLocalScopeRecords,
-                        out dynamicLocalMap,
-                        out dynamicLocalConstantMap,
+                    defaultNamespaceName = "";
+                }
+
+                hoistedLocalScopeRecords = ImmutableArray<HoistedLocalScopeRecord>.Empty;
+                dynamicLocalMap = null;
+                dynamicLocalConstantMap = null;
+                tupleLocalMap = null;
+                tupleLocalConstantMap = null;
+
+                byte[] customDebugInfo = symReader.GetCustomDebugInfoBytes(methodToken, methodVersion);
+                if (customDebugInfo != null)
```

Comment:
Will leave as is for now. There are already many methods here with long argument lists.